pkgs/by-name/js/jsoncons/package.nix

```diff
@@ -2,23 +2,25 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "jsoncons";
-  version = "0.176.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "danielaparker";
     repo = "jsoncons";
-    rev = "v${finalAttrs.version}";
-    hash = "sha256-RDF5HL6utM/6wna1TxCefl7X8B1qp6nsEDrguTLrtgA=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-mCiP6QdoM1MG1XRQ6cfKkkkZcSg6bxGcenMST3rHTFI=";
   };
 
   nativeBuildInputs = [ cmake ];
 
-  meta = with lib; {
+  env.NIX_CFLAGS_COMPILE = "-std=c++20 -Wno-error=deprecated-declarations";
+
+  meta = {
     description = "C++, header-only library for constructing JSON and JSON-like data formats";
     homepage = "https://danielaparker.github.io/jsoncons/";
-    changelog = "https://github.com/danielaparker/jsoncons/blob/${finalAttrs.src.rev}/CHANGELOG.md";
-    license = licenses.boost;
-    maintainers = with maintainers; [ sikmir ];
-    platforms = platforms.all;
+    changelog = "https://github.com/danielaparker/jsoncons/blob/${finalAttrs.src.tag}/CHANGELOG.md";
```

Comment:
You shouldn't depend on `src.tag` in `meta` (can't find the relevant umbrella issue for this).

Reply:
#367739